README.md

```diff
@@ -63,9 +64,15 @@ To enable linting, you'll need the general [linter](https://atom.io/packages/lin
 
 #### [Advanced New File](https://atom.io/packages/advanced-new-file)
 > Create multiple files and directories by typing a relative path.
+> Atom editor metioned that Advanced New File is no longer maintained for now. Please switch to [Advanced Open File](#advanced-open-file)
```

Comment:
Could you change this to:

    This package is currently unmaintained. It's advisable to to use [Advanced Open File](#advanced-open-file) instead.